com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -140,5 +137,13 @@ void Update()
                 QueueAssetLoading();
             }
         }
+
+#if UNITY_EDITOR
+        public void StripSupportData()
+        {
+            foreach (var asset in assets.Values)
+                asset.cellSupportDataAsset = null;
```

Comment:
Since OnBeforeSerialize reads this is serves as clearing out the use of the support data asset thus omitting it from builds.